### PR TITLE
Fix - this modal will make fun of you

### PR DESCRIPTION
**Pull request by gplanchat**
**Description**

When you don't configure any `pim_catalog_identifier` attribute (or your access rights don't grant you access to it in EE), you may face this situation, with a loader animation rolling over and over, never ending :

image

In fact, no action was taken when this case happened and the promise, once rejected, didn't have any error handling :

image

I added a quick error handling to help the poor integrators facing a broken demo to fix what has to be fixed in the installation files.

It probably needs another wording.

image

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | N/A
| Added Behats                      | N/A
| Changelog updated                 | No
| Review and 2 GTM                  | Yes
| Migration script                  | N/A
| Tech Doc                          | N/A

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/translations/jsmessages.en.yml [hunk 1]
```diff
@@ -56,6 +56,8 @@ flash:
         removed: Role successfully removed
 
 error:
+    creating:
+        product: There may not be any SKU attribute configured in the PIM, or you may not have the access rights to edit it.
```
Comment (willy-ahva) [inline src/Pim/Bundle/EnrichBundle/Resources/translations/jsmessages.en.yml]: "SKU" is already a name of an identifier attribute. Pls, change "SKU" to "identifier".

**Comment by fabienlem** [comment]
Hello Grégory!

Thank you a lot for this contribution :+1: 

I would be nice if you could add behat tests. Don't hesitate to ping us if you need help.

Also, I asked our product owners what could be the best error message. We will let you know as soon as possible.

Best regards,
Fabien

File: src/Pim/Bundle/EnrichBundle/Resources/translations/jsmessages.en.yml (hunk 1 above)
Comment (fabienlem) [inline src/Pim/Bundle/EnrichBundle/Resources/translations/jsmessages.en.yml]: Message provided by our product owners:
No attribute is configured as a product identifier or you don't have the rights to edit it.

**Comment by gplanchat** [comment]
@fabienlem concerning Behat tests, I'd like some hints to use the best approach. As long as it is a misconfiguration not present in the fixtures of the tests, I may not do the simplest thing.

**Comment by fabienlem** [comment]
Hello @gplanchat,

I discussed with the team, and we agree with you. We will take care of writing a test scenario but in the enterprise edition, this way we will be able to use the rights management in order to make the scenario easier.

One last thing, can you please replace the message for: "No attribute is configured as a product identifier or you don't have the rights to edit it."

Thank you!

Fabien

**Comment by fabienlem** [comment]
Hello @gplanchat!

Your PR has been merged in the master branch and I updated the 1.7 changelog.

Again, thank you for your contribution and for your reactivity :)

Best regards,
Fabien

**Pull request merged**